pkgs/tools/admin/proxmove/derivation.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "proxmove";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "ossobv";
+    repo = "proxmove";
+    rev = "v${version}";
+    sha256 = "sha256-8xzsmQsogoMrdpf8+mVZRWPGQt9BO0dBT0aKt7ygUe4=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    proxmoxer
+  ];
+
+  doInstallCheck = true;
+
+  preBuild = ''
+    rm -R assets
+    rm -R artwork
+  '';
+
+  installCheckPhase = ''
+    $out/bin/${pname} --version
+  '';
+
+  meta = with lib; {
+    description = "";
```

Comment:
Fill out description

Reply:
Whoops! That went up a bit fast. Thanks :)